packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -36,6 +36,9 @@ abstract class WebViewPlatformCallbacksHandler {
 
   /// Report web resource loading error to the host application.
   void onWebResourceError(WebResourceError error);
+
+  /// Invoked by [WebViewPlatformController] when a page requests authorization.
+  AuthInfo? onReceivedHttpAuthRequest(String host, String realm);
```

Comment:
This class is public API, so adding a new method without an implementation is a breaking change unless I'm missing something. This should probably have a default implementation that returns null.

Reply:
There is only one implementation in the `_PlatformCallbacksHandler` of the webview flutter widget. Where the new method is implemented. So it's not breaking anything, or?

Reply:
> There is only one implementation

If an interface is part of a public API surface, then you have no way of knowing how many implementations there are.

Reply:
ok added a null default